backend/python/app/graphql/queries/story_query.py

```diff
@@ -74,6 +82,7 @@ def resolve_story_translation_tests(
     )
 
 
+@require_authorization_by_role_gql({"User", "Admin"})
```

Comment:
this is actually not a query but a weird util function, you can remove this middleware call